recipes/ninjax/recipe.yaml

```diff
@@ -0,0 +1,42 @@
+context:
+  name: "ninjax"
+  version: "3.6.2"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: 3eabc014d94d05177f3ade206dc43a7503cb130eb6d396da73eb607b5c20054a
+
+build:
+  skip:
+    - win
```

Comment:
Looks like this one can ve noarch.

It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.